docroot/modules/humsci/hs_migrate/hs_migrate.module

```diff
@@ -71,7 +71,14 @@ function hs_migrate_entity_delete(EntityInterface $entity) {
   foreach ($migrations as $migration) {
     $destination = $migration->get('destination');
 
-    if (isset($destination['plugin']) && strpos($destination['plugin'], 'entity:') !== FALSE) {
+    if (!isset($destination['plugin'])) {
```

Comment:
Why would the destination plugin not be set?

Reply:
I'm trolling for a code comment here.

Reply:
It should always be set. but its just a safety valve